docs/swagger.yaml

```diff
@@ -0,0 +1,578 @@
+basePath: '{{.BasePath}}'
```

Comment:
maybe we only need one? swagger.json or swagger.yaml

Reply:
this is automatically generated by [gin-swagger](https://github.com/swaggo/gin-swagger), to keep only one we need to generate and delete the other. I think we can keep them both